src/Headless.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace HeadlessLaravel;
+
+use Composer\Autoload\ClassMapGenerator;
+use HeadlessLaravel\Cards\Card;
+use HeadlessLaravel\Cards\CardGroup;
+use HeadlessLaravel\Formations\Formation;
+use Illuminate\Support\Facades\Route;
+
+class Headless
+{
+    public function formations()
```

Comment:
I would like `Headless::formations()` to return a list of classes

Change this to `routeFormations()`